app/views/devise/registrations/_informable_fields.html.slim

```diff
@@ -22,7 +22,7 @@
       | Date Of Birth
       span.required *
     .col-md-8
-      = p.text_field :date_of_birth,class:"form-control"
+      = p.text_field :date_of_birth,id:'dob',class:"form-control"
```

Comment:
@GauravMakwana-BTC : use a space after , for better readability & use double quotes for string manipulation , use single quotes where we don't need to perform string interpolation